core/cli/src/error.rs

```diff
@@ -16,6 +16,9 @@
 
 //! Initialization errors.
 
+// Silence: `use of deprecated item 'std::error::Error::cause': replaced by Error::source, which can support downcasting`
+#![allow(deprecated)]
```

Comment:
if we use something deprecated don't we want to file a ticket for that and link it here and the other instances of this?